tutorials/bios-boot-tutorial/README.md

```diff
@@ -27,8 +27,8 @@ Minimal command line to launch the script below, make sure you replace `EOSIO_CO
 $ cd ~
 $ git clone https://github.com/EOSIO/eos.git
 $ cd ./eos/tutorials/bios-boot-tutorial/
-$ python3 bios-boot-tutorial.py --cleos="cleos" --nodeos=nodeos --keosd=keosd --contracts-dir="/EOSIO_CONTRACTS_DIRECTORY/" -a
+$ python3 bios-boot-tutorial.py --cleos=" --wallet-url http://127.0.0.1:6666 " --nodeos=nodeos --keosd=keosd --contracts-dir="EOSIO_CONTRACTS_DIRECTORY" -w -a
```

Comment:
Why non-default wallet location?

Reply:
Default wallet should work. Will change.